keyboards/themadnoodle/noodlepad_micro/readme.md

```diff
@@ -0,0 +1,30 @@
+# NoodlePad [Micro]
+
+![NoodPad[Mircro]](https://imgur.com/a/l1iZjWX)
+
+The Noodlepad [Micro] is a 6 key 2 encoder macro keypad designed using RP2040 chipset
+
+* Keyboard Maintainer: [The Mad Noodle](https://github.com/The-Mad-Noodle)
+* Hardware Supported: NoodlePad [Micro]
+* Hardware Availability: https://www.madnoodleprototypes.com/shop
+
+This folder is set as default in the parent rules.mk - it will build firmware compatible with the RP2040 Community Edition. See [here](https://docs.qmk.fm/#/platformdev_rp2040?id=rp2040_ce) for a list.
+
```

Comment:
```suggestion
```
This looks like remnants from some older text which is not relevant for this board.